### PR TITLE
Fix alerts and update their styling. Other minor changes

### PR DESCRIPTION
**Pull request by ConsM**
Addresses https://tyktech.atlassian.net/browse/TT-3340

Screenshots: 

Alert messages:
Screenshot 2021-09-10 at 13 11 45
Screenshot 2021-09-10 at 13 11 53

Buttons (Reject is red now) & some shadow to the header with CTAs
Screenshot 2021-09-10 at 14 16 07
Screenshot 2021-09-10 at 14 16 13

### CODE REVIEW

**Review by padiazg — APPROVED** [review]
LGTM

**Comment by ConsM** [comment]
Small update, have aligned the close icon vertically 

Screenshot 2021-09-13 at 08 29 11

**Review by cmelgarejo — APPROVED** [review]

**Comment by ConsM** [comment]
Added another commit to match the below.
Update tyk-ui.css to use the latest from our library

shadow-sticky-button

File: views/assets/stylesheets/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.qor-alert.qor-alert__active {
+  transform: translate(-50%,-110%);
+  display: flex;
+  margin:0 35px -60px 0;
+}
+
+
+.qor-alert--success {
+  background: #C2F2E9;
```
Comment (ifrim) [inline views/assets/stylesheets/components/alert.css:18]: Maybe put this in a var, or not.
The same below.

**Review by ifrim — APPROVED** [review]

**Pull request merged**